src/pages/index.js

```diff
@@ -24,10 +35,40 @@ newPopupImage.setEventListeners();
 const cardPopup = new PopupWithForm("#addModal", handleCardFormSubmit);
 cardPopup.setEventListeners();
 
-const newCardList = new Section(
-  { items: constants.initialCards, renderer: createCard },
-  ".cards__list"
+const avatarPopup = new PopupWithForm(
+  "#editAvatarModal",
+  handleAvatarFormSubmit
 );
+avatarPopup.setEventListeners();
+
+const popupAffirm = new PopupWithConfirm("#deleteCardModal");
+popupAffirm.setEventListeners();
+
+let newCardList;
+let card;
+
+api
+  .getInitialCards()
+  .then((res) => {
+    newCardList = new Section(
+      { items: res, renderer: createCard },
+      ".cards__list"
+    );
+    newCardList.renderItems();
+  })
+  .catch((err) => {
+    console.error(err);
+  });
+
+api
+  .getUserInfo()
+  .then((res) => {
+    useInfo.setUserInfo(res.name, res.about);
+    useInfo.setAvatar(res.avatar);
+  })
+  .catch((err) => {
+    console.error(err);
+  });
 
 document.addEventListener("DOMContentLoaded", function () {
   const addFormValidator = new FormValidator(
```

Comment:
**Could be improved**

You can **universally** create instances of validators for all forms in the project storing them inside one object `formValidators`.  And then you can take any validator using attribute `name` of the form where you need to disable/enable the submit button or remove errors.

```js
// define an object for storing validators
const formValidators = {}

const enableValidation = (config) => {
  const formList = Array.from(document.querySelectorAll(config.formSelector))
  formList.forEach((formElement) => {
    const validator = new FormValidator(config, formElement)
    // here you get the name of the form (if you don’t have it then you need to add it into each form in `index.html` first)
    const formName = formElement.getAttribute('name')

   // here you store the validator using the `name` of the form
    formValidators[formName] = validator;
    validator.enableValidation();
  });
};

enableValidation(config);

```

And now you can use them for disabling buttons or clearing errors:

```js

formValidators[ profileForm.getAttribute('name') ].resetValidation()

// or you can use a string – the name of the form (you know it from `index.html`)

formValidators['profile-form'].resetValidation()

```